quarkus/runtime/src/test/java/org/keycloak/quarkus/runtime/cli/PicocliTest.java

```diff
@@ -325,20 +335,30 @@ public void testReaugFromProdToDevExport() {
 
     @Test
     public void testNoReaugFromProdToExport() {
-        build("build");
+        build("build", "--db=dev-file");
 
         Environment.setRebuildCheck(); // will be reset by the system properties logic
-        NonRunningPicocli nonRunningPicocli = pseudoLaunch("export", "--file=file");
+        NonRunningPicocli nonRunningPicocli = pseudoLaunch("export", "--db=dev-file", "--file=file");
         assertEquals(CommandLine.ExitCode.OK, nonRunningPicocli.exitCode);
         assertFalse(nonRunningPicocli.reaug);
     }
 
+    @Ignore("Not valid until db is required for production")
```

Comment:
Can you please mention a GHI that tracks the breaking change we'll do in 27?

Reply:
Captured https://github.com/keycloak/keycloak/issues/35816 - put it as initially targetting 27.